addons/account/models/account_move.py

```diff
@@ -22,6 +22,13 @@ class AccountMove(models.Model):
     _order = 'date desc, id desc'
     _inherit = ['mail.thread', 'mail.activity.mixin']
 
+    @api.model
+    def default_get(self, fields):
+        rec = super(AccountMove, self).default_get(fields)
+        if not rec.get('journal_id'):
+            rec.update({'journal_id': self.env['account.journal'].search([('type', '=', 'general')], limit=1).id})
```

Comment:
use current company_id to find the journal.
@qdp-odoo

Reply:
@william-andre indeed. 

Can you also redo a valid commit message? I just ditched it by mistake while force-pushing the branch. Sorry about that

Reply:
@qdp-odoo done, and done